net.py

```diff
@@ -80,13 +107,13 @@ def build(
         config_filepath = '/etc/cloudcix/pod/configs/config.json'
 
     # netplan file
-    netplan_filepath = '/etc/netplan/00-installer-config.yaml'
+    netplan_filepath = f'/etc/netplan/{order}-{identifier}.yaml'
```

Comment:
I had thought the filename for the interface would be passed into the primitive? 

e.g. `O11-public0` is the naming convention in the current use case. This reads as if the primitive is designed for this case but not others that may consume it.

Reply:
okay then will replace `order` with `filename` should work.